src/test/claimtriebranching_tests.cpp

```diff
@@ -831,7 +831,7 @@ BOOST_AUTO_TEST_CASE(get_claim_by_id_test)
 {
     ClaimTrieChainFixture fixture;
     const std::string name = "test";
-    CMutableTransaction tx1 = fixture.MakeClaim(fixture.GetCoinbase(), name, "one", 1);
+    CMutableTransaction tx1 = fixture.MakeClaim(fixture.GetCoinbase(), name, "one", 2);
```

Comment:
Why is this changing?

Reply:
So the update can test the value stored at the index also gets updated from 2 to 1.